pom.xml

```diff
@@ -41,23 +41,13 @@
             <name>Mads Baggesen</name>
             <email>mads.baggesen@eficode.com</email>
         </developer>
-        <developer>
-            <id>gt_dbechtold</id>
-            <name>dbechtold</name>
-            <email>dbechtold@grammatech.com</email>
-        </developer>
-        <developer>
-            <id>aseno-grammatech</id>
-            <name>aseno</name>
-            <email>aseno@grammatech.com</email>
-        </developer>
     </developers>
 
     <scm>
-        <connection>scm:git:ssh://github.com/jenkinsci/codesonar-plugin.git</connection>
-        <developerConnection>scm:git:ssh://git@github.com/jenkinsci/codesonar-plugin.git</developerConnection>
-        <url>https://github.com/jenkinsci/codesonar-plugin</url>
-        <tag>HEAD</tag>
```

Comment:
connection URLs should not say eficode.

tag should say `HEAD`.

Looks like merge conflicts were not resolved correctly